swagger.go

```diff
@@ -270,12 +282,10 @@ window.onload = function() {
 	defaultModelsExpandDepth: {{.DefaultModelsExpandDepth}}
   })
 
-  const defaultClientId = "{{.Oauth2DefaultClientID}}";
-  if (defaultClientId) {
-    ui.initOAuth({
-      clientId: defaultClientId
-    })
-  }
+  ui.initOAuth({
+    clientId: {{.Oauth2DefaultClientID}},
+    usePkceWithAuthorizationCodeGrant: {{.UsePkceWithAuthorizationCodeGrant}}
```

Comment:
I hope this is backward compatible.

Reply:
No, it was not indeed. I changed it back. Calling Oauth2DefaultClientID will enable oauth2 and take the pkce property into account only then.